README.md

```diff
@@ -28,9 +28,10 @@ nx-parallel is a NetworkX backend that uses joblib for parallelization. This pro
 <details>
 <summary>Script used to generate the above list</summary>
   
-```.py
+```py
 import _nx_parallel as nxp
-d = nxp.get_funcs_info() # temporarily add `from .update_get_info import *` to _nx_parallel/__init__.py
+# temporarily add `from .update_get_info import *` to _nx_parallel/__init__.py
+d = nxp.get_funcs_info()
 for func in d:
     print(f"- [{func}]({d[func]['url']})")
```

Comment:
I don't think I agree with the suggestions that `prettier` is making here. They change the meaning of the comments and code. Is there a reason we are using prettier instead of black/ruff? 
```suggestion

# temporarily add `from .update_get_info import *` to _nx_parallel/__init__.py
for func in nxp.get_funcs_info():
    print(f"- [{func}]({d[func]['url']})")
```

Reply:
I agree, the suggestions by prettier are also removing the indentation. Also, we are using all 3- prettier, black and ruff. And I have made a temporary fix. And I will handle it in another PR(maybe [PR#67](https://github.com/networkx/nx-parallel/pull/67))